jobs/email-reminder/devops/vaults.json

```diff
@@ -8,10 +8,12 @@
     {
         "vault": "entity",
         "application": [
+            "postgres-legal",
             "filings-jobs",
             "nats-emailer",
             "entity-service-account",
-            "sentry"
+            "sentry",
+            "email-reminder"
```

Comment:
We need to create a new vault in 1pass `email-reminder` and add the below key into it:

- PAYMENT_SVC_FEES_URL: https://pay-api-dev.apps.silver.devops.gov.bc.ca/api/v1/fees